contrib/openbsm/bin/bsmconv/linau_event.c

```diff
@@ -0,0 +1,149 @@
+#include "linau_event.h"
+#include "linau_field.h"
+#include "linau_record.h"
+#include "linau_impl.h"
+#include "pjdlog.h"
+
+#include <stdlib.h>
+#include <stddef.h>
+
+#define	BSMCONV_LINAU_EVENT_ID_NVNAME		"id"
+#define	BSMCONV_LINAU_EVENT_TIMESTAMP_NVNAME	"timestamp"
+#define	BSMCONV_LINAU_EVENT_KEY_BUFFER		30
+
+linau_event *
+linau_event_create(void)
+{
+
+	return (linau_proto_create());
+}
+
+void
+linau_event_destroy(linau_event *event)
+{
+
+	PJDLOG_ASSERT(event != NULL);
```

Comment:
Assertions which check input arguments should be separated from the rest of a function with a single blank line.